libdnf/dnf-types.h

```diff
@@ -107,6 +107,7 @@ typedef enum {
         DNF_ERROR_CANNOT_WRITE_CACHE,                   /* Since: 0.7.0 */
         DNF_ERROR_NO_CAPABILITY,                        /* Since: 0.7.0 */
         DNF_ERROR_REMOVAL_OF_PROTECTED_PKG,             /* Since: 0.7.0 */
+        DNF_ERROR_UNKNOWN_OPTION,                       /* Since: 0.56.0 */
```

Comment:
The docstring needs to be updated for this.

Reply:
I added docstring for the new error code.

To be honest, I don't like this enum. Old explicitly assigned numbers look chaotic. And DNF_ERROR_LAST equals 55 (54 before), but DNF_ERROR_UNFINISHED_TRANSACTION equals 66. It look messy.
And we must be careful that the newly added error codes do not conflict with the existing explicitly defined ones.

Reply:
I agree with you on this, and this is something we should fix for libdnf v5.